hardware/opentrons_hardware/hardware_control/tool_sensors.py

```diff
@@ -85,11 +88,14 @@ async def capacitive_pass(
     mover: NodeId,
     distance: float,
     speed: float,
+    sensor_id: Optional[SensorId] = SensorId.S0,
 ) -> List[float]:
     """Move the specified axis while capturing capacitive sensor readings."""
     sensor_scheduler = SensorScheduler()
     sensor_info = SensorInformation(
-        sensor_type=SensorType.capacitive, sensor_id=SensorId.S0, node_id=tool
+        sensor_type=SensorType.capacitive,
+        sensor_id=sensor_id or SensorId.S0,
+        node_id=tool,
```

Comment:
To match making `sensor_id` non-`Optional`:

```suggestion
        sensor_type=SensorType.capacitive, sensor_id=sensor_id, node_id=tool
```